cypress/e2e/news.cy.js

```diff
@@ -0,0 +1,24 @@
+describe("Work & Co News links test", () => {
+  it("Opens W&C site/News, iterates and opens each link, checks H1 for each page", () => {
+    cy.visit("https://work.co/");
+    cy.get('[data-test-id="global-menu-btn"]').click();
```

Comment:
Selectors can be exported into fixtures. 
In general you would want to keep all your selectors in the fixtures.
Create data structure hierarchy. 
Top level sites > pages > elements. 

For example: 

sauce labs > home page > element x 
work.co > home page > element x

Reply:
As fixtures or as POM, as done for saucelabs for challenge 6?